drivers/SmartThings/matter-button/src/init.lua

```diff
@@ -281,19 +290,19 @@ local function initial_event_handler(driver, device, ib, response)
   end
 end
 
--- if the devce distinguishes a long press event, it will always be a "held"
+-- if the device distinguishes a long press event, it will always be a "held"
 -- there's also a "long release" event, but this event is required to come first
 local function long_press_event_handler(driver, device, ib, response)
   device:emit_event_for_endpoint(ib.endpoint_id, capabilities.button.button.held({state_change = true}))
-  local MSM = device:get_endpoints(clusters.Switch.ID, {feature_bitmap=clusters.Switch.types.SwitchFeature.MOMENTARY_SWITCH_MULTI_PRESS})
-  if contains(MSM, ib.endpoint_id) then
+  if get_field_for_endpoint(device, SUPPORTS_MULTI_PRESS, ib.endpoint_id) then
```

Comment:
I didn't see a reason it would recheck the feature flag rather than using the `SUPPORTS_MULTI_PRESS` field but let me know if I'm missing something.

Reply:
probably because initial drafts didn't set a field here, and I missed this one when that was changed